typescript-client/development/changelog.mdx

```diff
@@ -2,6 +2,19 @@
 title: Changelog
 ---
 
+# `0.0.511`
+
+### New Features
+
+This release introduces a new method to create attachments : `client.api.createAttachment`. This method is a wrapper around the `uploadFile` method and the `new Attachment` constructor. It allows you to create an attachment and attach it to a step in a single call.
```

Comment:
I would add the keyword `context-aware` method as you mentioned in the code, as in "we handle it for you, we make your life easier".

Reply:
yes, i changed it and mentioned both wrappers and context